pkgs/development/python-modules/torch/default.nix

```diff
@@ -335,7 +367,6 @@ in buildPythonPackage rec {
 
   checkPhase = with lib.versions; with lib.strings; concatStringsSep " " [
     "runHook preCheck"
-    cudaStubEnv
```

Comment:
Hm. Why does it need the LD_LIBRARY_PATH anymore?

Reply:
Setting `doCheck = true;`, it still works 🤷‍♂️

Should it not?

Reply:
I guess this makes sense, I'm not sure why the stub was ever needed (maybe some of binaries had libcuda.so in DT_NEEDED, and now they only do dlopen?)

Good riddance then